README.md

```diff
@@ -15,6 +15,11 @@ on how to do that, including how to develop and test locally and the versioning
 
 _Note: 1.28.0 and later require Gradle 7_
 
+### 2.1.1
+*Released*: TBD
```

Comment:
As always, please update the date before merge unless you don't need this change immediately.